bgpd/bgpd.c

```diff
@@ -2562,6 +2562,14 @@ int peer_group_remote_as(struct bgp *bgp, const char *group_name, as_t *as,
 	return 0;
 }
 
+int peer_notify_unconfig(struct peer *peer)
```

Comment:
I would request you to replace this function with MACRO :)

Reply:
just curious: why?

Reply:
I'm not certain a macro would be better than a function here -- is there a specific reason?

Reply:
The set of instructions in the code is not returning anything after execution to the called and  there is only one function being called inside this routine, only 3 lines of code.
So wanted to avoid the overhead of calling another function and reduce the function-call stack size. It's a suggestion for optimisation, even if we do-not do, it doesn't break anything :)

Reply:
We have lots of functions with a single line. For me personally, readability is better with a function rather than a macro.

Reply:
It is fine with me, if @riw777 is fine too. :)

Reply:
> We have lots of functions with a single line. For me personally, readability is better with a function rather than a macro.

I am fine with it, if @riw777 is fine too. :)